### PR TITLE
Some fixes for pyton Sentinel example

### PR DESCRIPTION
**Pull request by ghtmtt**
Making some attempts with the `openEO_Python_Sentinel2_EURAC.ipynb` files I noticed some errors in imports and other small stuff. This PR should fix the imports (when creating a clean py3 venv)

### CODE REVIEW

**Comment by soxofaan** [comment]
any reason you changed requirements.txt  that much? The set of dependencies of the python client should be as small as possible.

note that we are actually considering to drop requirements.txt: #74 

I think it would be better to have a requirements.txt file in the examples forlder, dedicated to the example notebooks

**Comment by ghtmtt** [comment]
> any reason you changed requirements.txt that much? The set of dependencies of the python client should be as small as possible.

I just tried to use the example in a clean py env but I had to pip install all the stuff in the requirements file

> note that we are actually considering to drop requirements.txt: #74

I'll suggest to keep a setup method for new comers so that they can easily try the examples. But of course this is just a suggestion

> I think it would be better to have a requirements.txt file in the examples forlder, dedicated to the example notebooks

that could be also an alternative. Please let me know if I can be helpful in some way. 

Thanks!

**Comment by soxofaan** [comment]
I added a requirements.txt file in the examples folder, as discussed above

File: examples/notebooks/openEO_Python_Sentinel2_EURAC.ipynb [hunk 1]
```diff
@@ -126,7 +126,7 @@
     "from matplotlib.colors import NoNorm\n",
     "from matplotlib_scalebar.scalebar import ScaleBar\n",
     "import numpy as np\n",
-    "from osgeo import gdal\n",
+    "import gdal\n",
```
Comment (soxofaan) [inline examples/notebooks/openEO_Python_Sentinel2_EURAC.ipynb]: as far as I understand `import gdal` is deprecated and `from osgeo import gdal` should be used instead

**Comment by soxofaan** [comment]
I also addressed the plot issue

I'll close this PR for now. Feel free to reopen if you want to further finetuning

**Pull request closed without merging**